### ISSUE TITLE
Zombie recovery

### ISSUE THREAD

**Issue by davetgc** [TheGameCreators/GameGuruRepo#2886]
Zombies stand back up when killed, but this needs to be defined by the user. A slider for a recovery chance is needed between 0 and 100%.

Also, a headshot should have a separate % slider.

E.g. You might create a zombie that stood back up 30% of the time, but would be killed by a headshot 100% of the time,

**Comment by fearlesswee** [comment]
To add onto this, zombies are also invulnerable when playing their "getting up" animation. I feel like this should not be the case.

**Comment by davetgc** [comment]
Yes, I agree with that one. It's odd you can't harm them as they stand up.
